cogs/get_token_authorisation.py

```diff
@@ -110,19 +126,15 @@ async def get_token_authorisation(self, ctx: TeXBotApplicationContext) -> None:
             user_name.text,
         )
 
-        # noinspection PyUnusedLocal
-        guest_role: discord.Role | None = None
-        with contextlib.suppress(GuestRoleDoesNotExistError):
-            guest_role = await ctx.bot.guest_role
+        MAKE_EPHEMERAL: bool = True
+
+        if "committee" in ctx.channel.name.lower():
```

Comment:
This is an insecure way of checking if the message should be ephemeral. The name of a channel has no bearing on whether the channel is actually only accessible to committee members. Although it would be foolish for users to name a channel containing committee yet allow others to read it, we have an obligation to ensure security despite the stupidity of users.

I would also argue that a channel that has permissions set to only allow committee to read it, yet does not have the word "committee" in the channel's name should be allowed to not have messages sent ephemerally, whereas this check would prevent it.

Reply:
I mean to a certain extent I think we could just remove the check entirely and leave the responsibility of determining whether it should be ephemeral or not up to the committee member running the command. Could be in the form of an argument or just two commands.